index.js

```diff
@@ -9,6 +9,8 @@ module.exports.add = (files) => {
 
 module.exports.amendCommit = () => execa('git', ['commit', '--amend', '--no-edit']);
 
+module.exports.branchName = () => execa('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
```

Comment:
Thanks for the contribution! Just a small suggestion, in case this project does other things with branch names in the future.

```suggestion
module.exports.getCurrentBranchName = () => execa('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
```